src/components/icon/icon.spec.ts

```diff
@@ -251,6 +253,53 @@ describe('Icon broadcast service', () => {
 
       expect(events.length).to.equal(0);
     });
+
+    it('when multiple broadcast services are initialized they should not send sync events to each other.', async () => {
+      const collections = createIconDefaultMap<string, SvgIcon>();
+      const references = createIconDefaultMap<string, IconMeta>();
+
+      const evt1 = [];
+      const broadcast1 = new IconsStateBroadcast(collections, references);
+      (broadcast1 as any).channel.addEventListener(
+        'message',
+        (e: MessageEvent<BroadcastIconsChangeMessage>) => {
+          evt1.push(e);
+        }
+      );
```

Comment:
Eh, not sure all of the extra broadcasts need to collect messages, the `channel` setup already does and will collect all other messages in `events` so you can check that collection instead of each extra broadcast :)